packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -136,35 +136,36 @@ export function reflects(
     value: any;
   }[]
 ): void {
-  it(`reflects`, async () => {
+  const cases = propsToTest.map(({ propertyName, value }) => [propertyName, value]);
+
+  it.each(cases)("%p", async (propertyName, value) => {
     const page = await simplePageSetup(componentTagOrHTML);
     const componentTag = getTag(componentTagOrHTML);
     const element = await page.find(componentTag);
 
-    for (const propAndValue of propsToTest) {
-      const { propertyName, value } = propAndValue;
-      const attrName = propToAttr(propertyName);
-      const componentAttributeSelector = `${componentTag}[${attrName}]`;
+    const attrName = propToAttr(propertyName);
+    const componentAttributeSelector = `${componentTag}[${attrName}]`;
 
-      element.setProperty(propertyName, value);
-      await page.waitForChanges();
+    console.log(componentAttributeSelector);
```

Comment:
remove console.log()?

Reply:
If I remove it, the whole test fails. 😂 Good catch.